src/lib.rs

```diff
@@ -101,7 +113,7 @@ impl<T> Webauthn<T> {
         Webauthn {
             // We use stdrng because unlike thread_rng, it's a csprng, which given
```

Comment:
Outdated comment?
`thread_rng` is cryptographically secure and it is what's being used.

Reply:
Fixed